homework-2.0/task7.rb

```diff
@@ -0,0 +1,50 @@
+class Validator
+  BRACKETS = {
+    '(' => ')',
+    '[' => ']',
+    '{' => '}'
+  }.freeze
+
+  def initialize(str)
+    @str = str
+    @stack = []
+  end
+
+  def valid?
+    return false if closer?(@str[0])
+
+    @str.split('').each do |chr|
+      if opener?(chr)
+        @stack.push(chr)
+      elsif closer?(chr)
+        if closes?(@stack.last, chr)
+          @stack.pop
+        else
+          return false
+        end
+      end
+    end
+    return true if @stack.empty?
```

Comment:
here if you have `@stack.empty?` true - you return true, if it's false you return false? 

I'd rather aboid such a conditions and use just @stack.empty as return value